tikv-configuration-file.md

```diff
@@ -1476,10 +1600,131 @@ raftdb 相关配置项。
 + 默认值：2
 + 最小值：1
 
+### `max-open-files`
+
++ RocksDB 可以打开的文件总数。
++ 默认值：`40960`
++ 最小值：`-1`
+
+### `max-manifest-file-size`
+
++ 单个 RocksDB Manifest 文件的最大大小。
++ 默认值：`"20MB"`
++ 最小值：`0`
++ 单位：B|KB|MB|GB
+
+### `create-if-missing`
+
++ 如果值为 `true`，当数据库不存在时将自动创建。
++ 默认值：`true`
+
+### `enable-statistics`
```

Comment:
FYI This is removed in https://github.com/pingcap/docs/pull/11626

Reply:
The configuration item is `raftdb.enable-statistics` and https://github.com/pingcap/docs/pull/11626 removes `rocksdb.enable-statistics`

Reply:
They should both be removed. Seems I forgot the raftdb one.

Reply:
https://github.com/pingcap/docs/pull/12665 PTAL